server/tmserver/world.py

```diff
@@ -656,6 +657,8 @@ def area_of_effect(cls, sender_obj):
 
     @classmethod
     def put_into(cls, outer_obj, inner_obj):
```

Comment:
in general using booleans to indicate success/failure is a code smell to me (eventually you might want a more nuanced report on error states than just true/false and it leads to an inconsistent API: usually state manipulation methods return `None`, like `remove_from`), and in similar situations here we're using `ClientException`. Is there a compelling reason to not raise `ClientException` if `outer_obj == inner_obj`?